src/ActionsMenuItem.scss

```diff
@@ -0,0 +1,30 @@
+.ActionsMenuItem__button {
+    width: 4rem;
+    float: right;
+    border: 0;
+    background-color: inherit;
+}
+
+.ActionsMenuItem__button-label {
+    margin-left: 0.5rem;
```

Comment:
Let's apply margin auto to all sides to place the label at the centre.

`
  .ActionsMenuItem__button-label {
    margin: auto;
}
`

Reply:
Done.